src/shells/webextension/backend.js

```diff
@@ -32,8 +32,11 @@ function handshake(hook, contentScriptId) {
     },
     send(data) {
       debugConnection('[BACKEND -> contentScript]', data);
+      
+      const payload = JSON.parse(JSON.stringify(data));
```

Comment:
This is quite a hacky solution, instead we should figure what data leads to this error and modify serializer https://github.com/mobxjs/mobx-devtools/blob/master/src/Bridge.js#L20 to handle this case